mail/templates/success/body.html

```diff
@@ -0,0 +1,19 @@
+{% extends "email_base.html" %}
+{% load static %}
+
+{% block content %}
+  <tr>
+      <td style="background-color: #ffffff;">
+        <p>Hello, your video is ready!</p>
+
+        <p>Title: <a href={{ video_url }}>{{video_title}}</a></p>
+        <p>Collection Title: <a href={{ collection_url }}>{{collection_title}}</a></p>
+        <p>Has finished processing and is now available on the OVS web site</p>
```

Comment:
```suggestion
```

I suggest moving this line up above.

Reply:
@pdpinch I made those changes and updated the screenshots

Reply:
@pdpinch do you want the signature on the error messages too or just the success email?

Reply:
It should be everywhere, but I'm happy with whatever is easiest.